### ISSUE TITLE
Python for Everyone, Interactive simple text issues

### ISSUE THREAD

**Issue by tstokke** [RunestoneInteractive/rs#451]
Moving my CS1 course to Runestone and I'm going through all the chapters. I noticed a couple of things that seem like they should be addressed. My apologies if this has been posted somewhere and I missed it, or if this should be somewhere else. 

- 1.5 shows <b>print</p>, where I assume you want **print** in bold. It does the same with one other word in the same window.

- In the Chapter 2 mixup questions, it didn't seem obvious that the code was supposed to be exactly what was done in the Parsons problems. I love the flow from solving the algorithm to having to actually type out the solution, but it wasn't obviously that was the expectation. But, a lot of simple things are not obvious to me, so I might be the problem. :-)

- In the Chapter 2 mixup questions, Activity 18, the expected text in the code is not the same as the text in the Parsons problem - "minutes" was added to the expected text. 

- In 4.7.3 Q-4 the expected answer is "exception". The way I read it "exceptions" would be the answer, or "an exception" would work. Grammar issues, it took me a while to figure out what it wanted. I actually had to peek at the source. I wonder if the regex could make the ending s optional?

Love what I'm seeing. If I come across anything else I'll update this (if I can), or add a new entry.

Tom

**Comment by pearcej** [comment]
@tstokke which base book are you using for CS1?

**Comment by tstokke** [comment]
Python for Everyone, Interactive

**Comment by pearcej** [comment]
Sorry. I guess I should have looked at the title. The repo you want is https://github.com/RunestoneInteractive/py4e-int

**Comment by pearcej** [comment]
Once you move them, could you possibly close them here? This repo handles runestone infrastrcture.

**Comment by tstokke** [comment]
Yeah, and here is where I run into problems. I'm not exactly sure how to proceed from here. :-( I have very limited experience with github, something that desperately needs to change.

**Comment by pearcej** [comment]
Just post your same issue on the issue queue, which is located here: https://github.com/barbarer/py4e-int/issues. Once you have done that, we can close the issue here.

**Comment by bnmnetp** [comment]
Closing here since these are all content fixes in the book itself rather than platform bugs — Python for Everyone (Interactive) is maintained at https://github.com/barbarer/py4e-int/issues, which is where the authors will see them.

@tstokke, sorry this stalled on the GitHub mechanics; if re-filing there is a hassle, the four items are recorded above and anyone can carry them over. Thanks for the careful read-through — the 4.7.3 "exception"/"exceptions" regex point in particular is a good catch.
